.github/workflows/main.yml

```diff
@@ -0,0 +1,43 @@
+# Taken from https://github.com/ssato/python-anyconfig/blob/next/.github/workflows/tests.yml
+# https://github.com/ssato/python-anyconfig/blob/next/LICENSE.MIT
+name: make
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    runs-on: ${{ matrix.platform }}
+    strategy:
+      max-parallel: 5
+      matrix:
+        platform:
+          - ubuntu-latest
+        python-version:
+          - 3.6
+          - 3.7
+          - 3.8
+          - 3.9
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: update package information
+        run: sudo apt-get -y -o APT::Immediate-Configure=false update
```

Comment:
What's the purpose of `-o APT::Immediate-Configure=false`?

Reply:
It is for avoiding unwanted interactions.
See also https://superuser.com/questions/199582/apt-error-could-not-perform-immediate-configuration-on.

Reply:
Ok, I have never seen this before but fine with me

Reply:
I added comments about the option.